acasclient/CmpdReg.py

```diff
@@ -80,8 +126,13 @@ def meta_lot_to_dict(meta_lot: dict) -> dict:
     # If lot amount is filled in, then we need to fill in lot barcode with the lot corp name
     lot_barcode = meta_lot["lot"]["corpName"] if meta_lot["lot"]["amount"] is not None else None
 
+    if use_parent_mol:
+        mol = meta_lot["lot"]["parent"]["molStructure"]
```

Comment:
Changed the default to use the parent mol structure because this is used for ETL purposes where we do not want lots to split off based on the way they were drawn if the destination server has different ACAS chemistry versions.

Reply:
That makes sense. It's hard to say, but using the parent structure seems like reasonable default behavior as most folks aren't even aware of the asDrawnStruct at the lot level